config.cfg

```diff
@@ -9,7 +9,14 @@ secret_key=YouMustChangeThisValueToSomething.
 token_duration=3600
 cors_origin=*
 bounding_box=MULTIPOLYGON (((-175 -12,-179.99999 -12,-179.99999 -20,-175 -20,-175 -12)), ((175 -12,179.99999 -12,179.99999 -20,175 -20,175 -12))) 
-require_auth=True
+require_auth=False
+
+[Database]

```

Comment:
we should pull this from the datacube.conf file see https://github.com/SatelliteApplicationsCatapult/helm-charts/blob/master/stable/cube-query/templates/worker-configmap.yaml for how it is deployed. Note we will need to change the helm charts to deploy the configuration to the server as well as the worker

Reply:
does the existing get_config function work with pulling this helm chart env out ?